graphql.go

```diff
@@ -178,8 +178,10 @@ func (c *Client) request(ctx context.Context, query string, variables map[string
 	}
 
 	if resp.StatusCode != http.StatusOK {
-		body, _ := io.ReadAll(resp.Body)
```

Comment:
Let's revert this. We should return the response body as a string.

Reply:
happy to add this back in